### PR TITLE
Estefanía Project react Native app

### PR DESCRIPTION
**Pull request by esteficodes**
My reflections on how this week's project turned out:
React native is quite complex to be learnt and handled in in just one week. Despite this, I was able to create the app I originally wanted to.

Things I'd like to have clarified or explained in more detail:
I think React Native would deserve at least two weeks in the programme. I feel like I would like to learn how to do so much more!

### CODE REVIEW

File: screens/HomeScreen.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      fetchAdvice();
+    },);
+
+    const API_URL='https://api.adviceslip.com/advice'
+  
+    const fetchAdvice = () => {
+      fetch(API_URL)
+      .then(res => res.json())
+      .then(data => setAdvice(data.slip.advice))
+      .catch(err => console.log(err));
```
Comment (KarolCislowski) [inline screens/HomeScreen.js:22]: remember to remove all console logs before requesting pull.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
I tried to run your app but I got this error: `Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.`

**Comment by esteficodes** [comment]
> I tried to run your app but I got this error: `Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.`

Hi, Karol! 
Could you be a little bit more specific as to what can I try here or what is exactly failing? I'm a little bit lost here.

**Review by KarolCislowski — APPROVED** [review]
Sweet, 😸  Now it works 🥳

**Pull request closed without merging**